cmd/neofs-cli/internal/client/client.go

```diff
@@ -148,21 +148,6 @@ func GetContainer(ctx context.Context, prm GetContainerPrm) (res GetContainerRes
 	return
 }
 
-// IsACLExtendable checks if ACL of the container referenced by the given identifier
-// can be extended. Client connection MUST BE correctly established in advance.
-func IsACLExtendable(ctx context.Context, c *client.Client, cnr cid.ID) (bool, error) {
```

Comment:
And this one as well. Logically it's the same change (you get container earlier, so this code is no longer needed).